docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -134,10 +139,31 @@ To run in HA mode, launch multiple Alluxio masters, point them to a shared journ
 and set their Zookeeper configuration.
 
 ```bash
-docker run -d --net=host \
+docker run -d \
            ...
-           -e ALLUXIO_MASTER_JOURNAL_FOLDER=hdfs://[namenodeserver]:[namenodeport]/alluxio_journal
-           -e ALLUXIO_ZOOKEEPER_ENABLED=true -e ALLUXIO_ZOOKEEPER_ADDRESS=zkhost1:2181,zkhost2:2181,zkhost3:2181 \
+           -e ALLUXIO_JAVA_OPTS="-Dalluxio.master.embedded.journal.addresses=master-hostname-1:19200,master-hostname-2:19200,master-hostname-3:19200 -Dalluxio.master.hostname=master-hostname-1 \
```

Comment:
since docker runs just a single service, I don't think there's a need to differentiate between master/worker java opts. Anyway, all the options here apply equally to both master and worker